contrib/samples/openvino/deployer/component.yaml

```diff
@@ -0,0 +1,55 @@
+name: OpenVINO model server deployer
+description: Deploys OpenVINO Model Server instance to Kubernetes and runs model evaluation
+inputs:
+- name: Batch size
+  default: auto
+- name: Log level
+  default: DEBUG
+- name: Model export path
+  default: 'gs://intelai_public_models/resnet_50_i8'
+- name: Model version policy
+  default: '{"latest": { "num_versions":2 }}'
+- name: Replicas
+  default: '1'
+- name: Server name
+  default: resnet
+- name: Evaluation images list
+  default: 'https://raw.githubusercontent.com/IntelAI/OpenVINO-model-server/master/example_client/input_images.txt'
+- name: Image path prefix
+  default: 'https://github.com/IntelAI/OpenVINO-model-server/raw/master/example_client/'
+- name: Model input name
+  default: 'data'
+- name: Model output name
+  default: 'prob'
+- name: Model input size
+  default: 224
+outputs:
+- name: Server endpoint
+metrics:
+  - name: latency
+  - name: accuracy
+implementation:
```

Comment:
Kubeflow components do not support multiple containers in one component. Best to break these out into three components.

Reply:
Also "metrics" attribute has never existed.